### PR TITLE
Add Cloud Function to fetch account <-> wallet address mapping

### PR DESCRIPTION
**Pull request by gnardini**
### Description

Added a Cloud Function that allows us to fetch the account address(es) that are linked to a wallet address. This will be used in a tool that will allow users to transfer funds from their MTW to their EOA if they sent funds to the MTW my mistake. This happened to some users recently because of a bug in the WalletConnect integration.

### Other changes

N/A

### Tested

Manually using the tool mentioend above

### How others should test

If you need access to the tool let me know, but this shouldn't be a part of our QA release testing

### Related issues

- See https://valora-app.slack.com/archives/C01D6H6116V/p1630542810004200

### Backwards compatibility

N/A

### CODE REVIEW

File: packages/cloud-functions/README.md [hunk 1]
```diff
@@ -0,0 +1,5 @@
+# Cloud Functions
+
+This package contains Cloud Functions that are deployed in Firebase.
+
+Before deploying you need to make sure the `yarn.lock` file is up to date. To do that, you need to put  `"packages/!(cloud-functions)*"` in the `workspaces.packages` section of the root `package.json` temporarily. Once you did that, you can just run `yarn` in this directory to update the `yarn.lock`. Remember to undo the changes in the root `package.json` after doing this!
```
Comment (jeanregisser) [inline packages/cloud-functions/README.md:5]: 👍

File: packages/cloud-functions/src/walletAddressMapping/index.ts [hunk 2]
```diff
@@ -0,0 +1,23 @@
...
+export const fetchAccountsForWalletAddress = functions.https.onRequest(async (req, res) => {
+  try {
+    res.set('Access-Control-Allow-Origin', '*')
+    const walletAddress = req.query.walletAddress as string
+    if (!walletAddress) {
+      res.status(400).end()
+      return
+    }
+    await initDatabase()
+    const accountAddresses = await knexDb('account_wallet_mappings')
```
Comment (jeanregisser) [inline packages/cloud-functions/src/walletAddressMapping/index.ts:13]: Who updates this table?

**Review by jeanregisser — APPROVED** [review]
Thanks 👍

File: packages/cloud-functions/src/walletAddressMapping/index.ts (hunk 2 above)
Comment (gnardini) [reply packages/cloud-functions/src/walletAddressMapping/index.ts:13]: The [indexer](https://github.com/valora-inc/indexer/)

**Pull request merged**